python/py-nbxmpp/Portfile

```diff
@@ -4,14 +4,14 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-nbxmpp
-version             0.6.10
+version             2.0.6
 revision            0
 categories-append   net
 platforms           darwin
 license             GPL-3
 supported_archs     noarch
 
-python.versions     27 37 38 39
+python.versions     27 37 38 39 310
```

Comment:
the version you're updating to does not support PY27, so you should remove this here or if `py27-nbxmpp` is a dependency of something else you need to restrict that subport to the latest supported version.